in/command_test.go

```diff
@@ -372,6 +372,167 @@ var _ = Describe("In Command", func() {
 				})
 			})
 		})
+
+		Context("when the requested path is the initial path", func() {
+			var initialFilename string
+
+			BeforeEach(func() {
+				initialFilename = "a-file-0.0"
+				request.Source.InitialPath = "files/a-file-0.0"
+				request.Version.Path = request.Source.InitialPath
+				request.Source.InitialContentText = "the hard questions are hard 🙈"
+			})
+
+			It("it creates a file containing the initial text content", func() {
+				_, err := command.Run(destDir, request)
+				Ω(err).ShouldNot(HaveOccurred())
```

Comment:
The existing tests in this file use the `Expect(foo).To(Bar())` style of gomega assertions. We should keep to that style for consistency.

Actually now that I read more closely, I see that there's a complete mixture of the 2 styles in both this file, and the repo. Joy. I'm not sure what the best thing to do is here...

Reply:
😞 Indeed, I chose not to point this out when pairing for that reason.